execution-plan/src/api_request.rs

```diff
@@ -89,6 +89,47 @@ impl ApiRequest {
                 log_req(events);
                 session.run_command(cmd_id, ModelingCmd::from(cmd)).await?
             }
+            Endpoint::ImportFiles => {
+                let mut args_iter = arguments;
+                let arg_opt_import_files_struct = args_iter.next();
+
+                let Some(arg_import_files_struct_prim) = arg_opt_import_files_struct else {
```

Comment:
Every other API request's parameters get instantiated via `each_cmd::CommandName::from_memory)` call. Why not just do that here?

Reply:
I was not able to use `from_memory` with this specific structure successfully